File: 0455-assign-cookies/0455-assign-cookies.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);

        int cnt = 0;
        int i = 0;
        int j = 0;

        while(i < g.length && j < s.length){
            if(g[i] <= s[j]){
                cnt++;
                i++;
            }
            
            j++;
        }

        return cnt;
    }
}

File: 0455-assign-cookies/0455-assign-cookies.py
class Solution(object):
    def findContentChildren(self, g, s):
        g.sort()
        s.sort()
        i, j = 0, 0
        count = 0
        
        while i < len(g) and j < len(s):
            if g[i] <= s[j]:
                count += 1
                i += 1
            j += 1
        
        return count
